script6.js

```diff
@@ -0,0 +1,27 @@
+/* 6.1 */
+
+let Cars = [];
```

Comment:
It is convention to use camelCase to name variables, so this could be named `cars`

Reply:
Thanks I get it. I will correct.